lib/util.js

```diff
@@ -30,6 +30,9 @@ module.exports = {
   decorateRouteWithPrerequisites (app, route) {
     const policies = app.config.policies
     const prerequisites = lib.Util.getRoutePrerequisites(policies, route)
+    if (!_.isArray(prerequisites)) {
+      app.log.error('trailpack-router: policy', prerequisites, 'must be an array.')
+    }
```

Comment:
I think we should return here, right? Otherwise, the app will still try to use the wrong value. I think we want to ignore this policy config, and throw the error